src/frontend/web/src/ratingpage/RatingButton.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'antd/lib/button';
+import { DislikeOutlined, LikeOutlined } from '@ant-design/icons';
+import './RatingButton.scss';
+
+const RatingButton = ({ className, icon, onClick, flashFunc }) => {
+    const onClickHandler = () => {
+        flashFunc();
+        // Wait until the flash animation disappears.
+        setTimeout(() => { onClick(); }, 900);
+    };
+
+    const iconSelector = (icon === 'DislikeOutlined')
+        ? <DislikeOutlined style={{ fontSize: '7vh' }} />
+        : <LikeOutlined style={{ fontSize: '7vh' }} />;
+
```

Comment:
We need to locate the buttons relative to the page and not in an absolute reference.
When I minimize the page the buttons cover the card.